test/e2e/dynamic_provisioning_test.go

```diff
@@ -493,6 +493,35 @@ func (t *dynamicProvisioningTestSuite) defineTests(isMultiZone bool) {
 		}
 		test.Run(cs, ns)
 	})
+
+	ginkgo.It("should create a volume azuredisk with tag [kubernetes.io/azure-disk] [disk.csi.azure.com] [Windows]", func() {
+		skipIfTestingInWindowsCluster()
+		pods := []testsuites.PodDetails{
+			{
+				Cmd: convertToPowershellCommandIfNecessary("while true; do echo $(date -u) >> /mnt/test-1/data; sleep 3600; done"),
+				Volumes: t.normalizeVolumes([]testsuites.VolumeDetails{
+					{
+						ClaimSize: "10Gi",
+						MountOptions: []string{
+							"barrier=1",
+							"acl",
+						},
+						VolumeMount: testsuites.VolumeMountDetails{
+							NameGenerate:      "test-volume-",
+							MountPathGenerate: "/mnt/test-",
+						},
+					},
+				}, isMultiZone),
+				IsWindows: isWindowsCluster,
+			},
+		}
+		test := testsuites.DynamicallyProvisionedAzureDiskWithTag{
+			CSIDriver:              testDriver,
+			Pods:                   pods,
+			StorageClassParameters: map[string]string{"skuName": "Standard_LRS", "tags": "tag=test"},
```

Comment:
may pass this `"tag=test"` as a field member of `DynamicallyProvisionedAzureDiskWithTag` and could verify this var inside `DynamicallyProvisionedAzureDiskWithTag`